Bluepill-cli/BPInstanceTests/BluepillTests.m

```diff
@@ -344,6 +344,137 @@ - (void)testRunWithPassingTestsSet {
     XCTAssert(exitCode == BPExitStatusTestsAllPassed);
 }
 
+- (void)testReuseSimulator {
+    //[BPUtils quietMode:NO];
```

Comment:
If we move this to setup as default, then there will be too much output from every test case.
ideally, I think should have different control when we run the tests in different environments, like CI vs local. So the hardcoded way is not the good way. Better way should be controlled by an environment variable or CLI option passed to test (not sure how to do) .
As @Ashit said, our quite and debug mode in the current code is a mess. So I think I will just leave these few lines of commented code in the test cases for now, and do a clean up when we have sorted out a better way to control quite and debug mode.